app/repos/breaches.js

```diff
@@ -11,14 +12,23 @@ const { BreachCategory } = require('../data/domain')
  *
  * @return {Promise<BreachCategoryDao[]>}
  */
-const getBreachCategoryDAOs = async () => {
+const getBreachCategoryDAOs = async (userSelectableOnly = false) => {
+  const where = userSelectableOnly
```

Comment:
What was the reasoning for a 'user-selectable' column?

Reply:
otherwise the two new fields would show on the multiple select page